*** outs/double_quotes_tests.sh_bash.out	2024-12-03 22:19:01.625540213 +0100
--- outs/double_quotes_tests.sh_minishell.out	2024-12-03 22:19:02.301542020 +0100
***************
*** 1,9 ****
  hello    wooorld
  hel|lo   ;wooorld;
  h\el\|lo   \;woo\orld;
  'yoooo'
  'yoooo\'
! 'yoooo tok\en\5'"
  one      big     token
  one      big     tokenhello
  hione      big     tokenhello
--- 1,19 ----
+ minishell: /bin/echo "hello    wooorld"
  hello    wooorld
+ minishell: /bin/echo "hel|lo   ;wooorld;"
  hel|lo   ;wooorld;
+ minishell: /bin/echo "h\el\|lo   \;woo\orld;"
  h\el\|lo   \;woo\orld;
+ minishell: /bin/echo "'yoooo'"
  'yoooo'
+ minishell: /bin/echo "'yoooo\'"
  'yoooo\'
! minishell: /bin/echo "'yoooo tok\en\5'\""
! minishell: /bin/echo "one      big     token"
  one      big     token
+ minishell: /bin/echo "one      big     token""hello"
  one      big     tokenhello
+ minishell: /bin/echo hi"one      big     token"hello
  hione      big     tokenhello
+ minishell: exit 0
+ exit
