*** outs/env_tests.sh_bash.out	2024-12-03 22:19:03.409544983 +0100
--- outs/env_tests.sh_minishell.out	2024-12-03 22:19:04.801548706 +0100
***************
*** 1,14 ****
  hi
! hi tmatis
! hi tmatis
! declare -x var1="hi tmatis"
  declare -x var_unset
  
  set now
  var_unset=set now
  
! yoyo
  yo
  yo echo log
  var6=yo
  declare -x noassign
--- 1,39 ----
+ minishell: export var1=hi
+ minishell: echo $var1
  hi
! minishell: export var1+=" tmatis"
! minishell: echo $var1
! hi
! minishell: export var1
! minishell: echo $var1
! hi
! minishell: export | grep var1
! declare -x var1="hi"
! minishell: export var_unset
! minishell: export | grep var_unset
  declare -x var_unset
+ minishell: echo $var_unset
  
+ minishell: export var_unset="set now"
+ minishell: echo $var_unset
  set now
+ minishell: env | grep var_unset
  var_unset=set now
+ minishell: export var_unset=
+ minishell: echo $var_unset
  
! minishell: export var2=yo var2+=yo
! minishell: echo $var2
  yo
+ minishell: export var5=yo; echo $var5
+ minishell: export var6=yo var7=log
+ minishell: echo $var6 echo $var7
  yo echo log
+ minishell: env | grep var6
  var6=yo
+ minishell: export noassign
+ minishell: export | grep noassign
  declare -x noassign
+ minishell: env | grep noassign
+ minishell: exit 0;
+ exit
