*** outs/pipes_tests.sh_bash.out	2024-12-03 22:19:08.737559230 +0100
--- outs/pipes_tests.sh_minishell.out	2024-12-03 22:19:09.613561573 +0100
***************
*** 1,3 ****
--- 1,4 ----
+ minishell: cat test.sh | cat -e
  # **************************************************************************** #$
  #                                                                              #$
  #                                                         :::      ::::::::    #$
***************
*** 115,124 ****
  exit $error$
  $
  #rm -r ./outs/$
  1
! 2
  10
  0
  # **************************************************************************** #
  #                                                                              #
  #                                                         :::      ::::::::    #
--- 116,131 ----
  exit $error$
  $
  #rm -r ./outs/$
+ minishell: echo hello | wc -l
  1
! minishell: export | grep USER | wc -l
! 4
! minishell: cat /dev/urandom | head -n 10 | wc -l
  10
+ minishell: cd ./sandbox
+ minishell: cat ../test.sh > file | cat -e > file2 | wc -l
  0
+ minishell: cat file file2
  # **************************************************************************** #
  #                                                                              #
  #                                                         :::      ::::::::    #
***************
*** 236,239 ****
--- 243,249 ----
  exit $error
  
  #rm -r ./outs/
+ minishell: echo hello | cat < file | wc
      117     398    4073
+ minishell: exit 0
+ exit
