*** outs/unset_tests.sh_bash.out	2024-12-03 22:19:14.113573606 +0100
--- outs/unset_tests.sh_minishell.out	2024-12-03 22:19:15.097576237 +0100
***************
*** 1,2 ****
--- 1,16 ----
+ minishell: export hello=yo
+ minishell: export | grep hello
  declare -x hello="yo"
+ minishell: env | grep hello
  hello=yo
+ minishell: unset hello
+ minishell: export | grep hello
+ minishell: env | grep hello
+ minishell: export var1
+ minishell: unset var1
+ minishell: export var2=yolo var3=yolo
+ minishell: unset var2 var3
+ minishell: export | grep var2
+ minishell: export | grep var3
+ minishell: exit 0
+ exit
